aspnetcore/getting-started.md

```diff
@@ -25,20 +25,34 @@ uid: getting-started
     dotnet new webapp -o aspnetcoreapp
     ```
 
-3. Run the app with the following commands:
+3. Trust the HTTPS development certificate:
+
+    ```terminal
+    dotnet dev-certs https --trust
```

Comment:
Note this only works on Windows and macOS. On Linux you have to manually do distro specific steps to trust the certificate.